### PR TITLE
fix issue #993

### PR DESCRIPTION
**Pull request by TaisukChoi**
Fixed issue #993 by setting `message_column` to be optional and adding to description that either `message_column` or `content_template_id` is required.

### CODE REVIEW

File: genesyscloud/outbound/resource_genesyscloud_outbound_messagingcampaign.go [hunk 1]
```diff
@@ -51,8 +51,8 @@ var (
 	outboundmessagingcampaignsmsconfigResource = &schema.Resource{
 		Schema: map[string]*schema.Schema{
 			`message_column`: {
-				Description: `The Contact List column specifying the message to send to the contact.`,
-				Required:    true,
+				Description: `The Contact List column specifying the message to send to the contact. Either message_column or content_template_id is required.`,
```
Comment (dginty4) [inline genesyscloud/outbound/resource_genesyscloud_outbound_messagingcampaign.go:54]: Probably best to add the same message to `content_template_id`

File: go.mod
[the remark this answers is not in the document]
Comment (TaisukChoi) [reply go.mod:1]: I removed changes to go.sum & go.mod.

File: genesyscloud/outbound/resource_genesyscloud_outbound_messagingcampaign.go [hunk 2]
```diff
@@ -66,7 +66,7 @@ var (
 				Type:        schema.TypeString,
 			},
 			`content_template_id`: {
-				Description: `The content template used to formulate the message to send to the contact.`,
+				Description: `The content template used to formulate the message to send to the contact. Either message_column or content_template_id is required.`,
```
Comment (HemanthDogiparthi12) [inline genesyscloud/outbound/resource_genesyscloud_outbound_messagingcampaign.go:69]: Will this work , if either of the two attributes are not provided ? If no, please add a validate function to check either of the two params is present.

Comment (charliecon) [reply genesyscloud/outbound/resource_genesyscloud_outbound_messagingcampaign.go:69]: Also, you can specify that both cannot be set together using the `ConflictsWith` array here

```golang
ConflictsWith: []string{"message_column"}
```

**Review by charliecon — APPROVED** [review]

**Comment by charliecon** [comment]
Hi @TaisukChoi 

We no longer merge directly into `main` - we merge into `dev` first. You'll need to merge `MyPureCloud:dev` into your branch, fix any conflicts, and then change the target branch of this PR to dev before we can merge. If you have any questions at all, don't hesitate

Thanks

**Review by carnellj-genesys — APPROVED** [review]
Looks good.  Thanks for contributing

**Comment by charliecon** [comment]
> Is this guy ready to be merged

@carnellj-genesys Not yet. See me comment above about changing the target branch and merging the latest version of `dev` to fix conflicts. 

@TaisukChoi Since they are small changes, you could re-do the fork off of dev and implement the changes again, if you think it would be less work.

**Comment by TaisukChoi** [comment]
> > Is this guy ready to be merged
> 
> @carnellj-genesys Not yet. See me comment above about changing the target branch and merging the latest version of `dev` to fix conflicts.
> 
> @TaisukChoi Since they are small changes, you could re-do the fork off of dev and implement the changes again, if you think it would be less work.

@charliecon  Yes. I plan to fork off dev, add ConflictsWith and possibly validation functions and request merge again.

**Pull request closed without merging**